Triangulation/include/CGAL/Triangulation.h

```diff
@@ -437,15 +448,28 @@ class Triangulation
 
     Full_cell_const_iterator full_cells_begin() const { return tds().full_cells_begin(); }
     Full_cell_const_iterator full_cells_end()   const { return tds().full_cells_end(); }
-
+#if 0
     Finite_full_cell_iterator finite_full_cells_begin()
     { return Finite_full_cell_iterator(Finiteness_predicate(*this), full_cells_begin(), full_cells_end()); }
     Finite_full_cell_iterator finite_full_cells_end()
     { return Finite_full_cell_iterator(Finiteness_predicate(*this), full_cells_end(), full_cells_end()); }
+
     Finite_full_cell_const_iterator finite_full_cells_begin() const
     { return Finite_full_cell_const_iterator(Finiteness_predicate(*this), full_cells_begin(), full_cells_end()); }
     Finite_full_cell_const_iterator finite_full_cells_end() const
     { return Finite_full_cell_const_iterator(Finiteness_predicate(*this), full_cells_end(), full_cells_end()); }
+#else
+/*
```

Comment:
@sloriot When I uncomment the non-const version I get a compilation error. Can you explain why?

Reply:
Fixed. I don't get why I need to define extra operators. I would have expect that there would be an implicit conversion then.